sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_events_v1/query.sql

```diff
@@ -154,343 +263,258 @@ client_product_feature_usage AS (
   LEFT JOIN
     UNNEST(event_extra) AS extra
   WHERE
-    DATE(submission_timestamp) = @submission_date
+    DATE(submission_timestamp)
+    BETWEEN DATE_SUB(@submission_date, INTERVAL 4 DAY)
+    AND @submission_date
+    AND DATE(
+      DATETIME(LEAST(ping_info.parsed_start_time, ping_info.parsed_end_time), 'UTC')
+    ) = DATE_SUB(@submission_date, INTERVAL 4 DAY)
   GROUP BY
-    client_info.client_id,
-    submission_date
-),
-product_features_agg AS (
-  SELECT
-    submission_date,
+    ping_date,
+    client_id,
+    channel,
+    country
+)
+SELECT
+  ping_date,
```

Comment:
Please also add `@submission_date AS submission_date` before the `ping_date` field, we will need this for partitioning.

Reply:
Please also make sure to include it in the group by below.